common/featureset.cpp

```diff
@@ -21,6 +21,7 @@ void FeatureSet::initalizeFeatureList(QMap<QString, bool> &featureList) {
     featureList.insert("client_warnings", false);
     featureList.insert("mod_log_lookup", false);
     featureList.insert("client_inactivetimeout", false);
+    featureList.insert("donator_recognition", false);
```

Comment:
Of course , that is unless we intend on creating a new user level with this type of change.  I don't think I would recommend that but maybe then it should be in the featureset list?